src/shared/Atlassian.Bitbucket/BitbucketHostProvider.cs

```diff
@@ -328,6 +411,59 @@ private async Task<bool> RequiresTwoFactorAuthenticationAsync(ICredential creden
             }
         }
 
+        private async Task<bool> ValidateCredentialsWork(InputArguments input, ICredential credentials, AuthenticationModes authModes) 
+        {
+            if(credentials == null)
+            {
+                return false;
+            }
+
```

Comment:
Could add a `TODO` here that ideally we'd also check if the credentials have expired based on some local metadata (once/if we get such metadata storage), and return `false` if they have. Just for future readers :)